docs/B-Computations/expressions.md

```diff
@@ -1,5 +1,9 @@
 ---
+id: expressions
+title: Expressions
 sidebar_position: 3
+slug: /B-computations/expressions
```

Comment:
The slug should follow the standard format `/computations/expressions` as per https://github.com/Seneca-ICTOER/IPC144/issues/64